tests/tests/checks/snapshots/analysis/js/export.mjs/check_glob@exported_decls__json.snap

```diff
@@ -0,0 +1,255 @@
+---
+source: tests/test_check_insta.rs
+expression: "&json_results"
+---
+[
+  {
+    "loc": "00001:0",
+    "target": 1,
+    "kind": "def",
+    "pretty": "file js/export.mjs",
+    "sym": "FILE_js/export@2Emjs"
+  },
+  {
+    "loc": "1:11-18",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property letDecl",
+    "sym": "#letDecl"
+  },
+  {
+    "loc": "1:11-18",
+    "target": 1,
+    "kind": "def",
+    "pretty": "letDecl",
+    "sym": "#letDecl",
+    "context": ""
+  },
+  {
+    "loc": "2:13-22",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property constDecl",
+    "sym": "#constDecl"
+  },
+  {
+    "loc": "2:13-22",
+    "target": 1,
+    "kind": "def",
+    "pretty": "constDecl",
+    "sym": "#constDecl",
+    "context": ""
+  },
+  {
+    "loc": "3:16-23",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property funDecl",
+    "sym": "#funDecl",
+    "nestingRange": "3:26-3:27"
+  },
+  {
+    "loc": "3:16-23",
+    "target": 1,
+    "kind": "def",
+    "pretty": "funDecl",
+    "sym": "#funDecl",
+    "context": ""
+  },
+  {
+    "loc": "4:13-22",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property classDecl",
+    "sym": "#classDecl",
+    "nestingRange": "4:22-4:24"
+  },
+  {
+    "loc": "4:13-22",
+    "target": 1,
+    "kind": "def",
+    "pretty": "classDecl",
+    "sym": "#classDecl",
+    "context": ""
+  },
+  {
+    "loc": "5:4-9",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property dummy",
+    "sym": "#dummy"
+  },
+  {
+    "loc": "5:4-9",
+    "target": 1,
+    "kind": "def",
+    "pretty": "dummy",
+    "sym": "#dummy",
+    "context": ""
+  },
+  {
+    "loc": "5:15-21",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property local1",
+    "sym": "#local1"
+  },
+  {
+    "loc": "5:15-21",
+    "target": 1,
+    "kind": "def",
+    "pretty": "local1",
+    "sym": "#local1",
+    "context": ""
+  },
+  {
+    "loc": "5:27-33",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property local2",
+    "sym": "#local2"
+  },
+  {
+    "loc": "5:27-33",
+    "target": 1,
+    "kind": "def",
+    "pretty": "local2",
+    "sym": "#local2",
+    "context": ""
+  },
+  {
+    "loc": "6:9-15",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property local1",
+    "sym": "#local1"
+  },
+  {
+    "loc": "6:9-15",
+    "target": 1,
+    "kind": "def",
+    "pretty": "local1",
+    "sym": "#local1",
+    "context": ""
+  },
+  {
+    "loc": "6:17-23",
+    "source": 1,
+    "syntax": "def,prop",
+    "pretty": "property local2",
+    "sym": "#local2"
+  },
+  {
+    "loc": "6:17-23",
+    "target": 1,
+    "kind": "def",
+    "pretty": "local2",
+    "sym": "#local2",
+    "context": ""
+  },
```

Comment:
I wonder if we should either use the "alias" kind (introduced for [typedefs/using aliases](https://searchfox.org/mozilla-central/rev/6d437ba43e7752a85f5731133ee66152b390aa16/build/clang-plugin/mozsearch-plugin/MozsearchIndexer.cpp#1817)) or, potentially better, explicitly define import/export kinds.  Specifically, it's our policy to only generate "go to def" context menu links when there is a single def, so having exports treated as defs is suboptimal for that purpose, although we tend to have so many collisions in JS because of the single global namespace soup we put everything in, I don't know that it matters that much.

No need to change anything in this patch, but I think it's a potential interesting refinement.  (And arguably it's better to do in a different patch/stack since it involves a number of changes to things like router.py.  I imagine the patch that added "alias" probably would provide useful context.)

Reply:
I'll experiment with "alias" in followup